eng/CredScanSuppression.json

```diff
@@ -25,6 +25,42 @@
       "hash": "5ETIzLYe9aa9Xx5512Uy6gJiCZ/CL2QzS3nDfN51nz8=",
       "_justification": "Secret used by test code"
     },
+    {
+      "hash": "UyIHw8OdxjE4j2ChZgotLVNB7fRelDvB2QGJC4Ywhgg=",
```

Comment:
These type of suppressions are not stable based on other changes in the file. Is there another strategy we can use. Such as using a placeholder or a refactored file that only contains test secrets that we can add to the suppression file list.

Reply:
yeah that's right! We already have some files holding test secret constants. I added some more. Not suitable for all these cases though. will think more about the rest of hashes.